apps/api/package.json

```diff
@@ -20,6 +20,7 @@
     "start:build": "TZ=UTC node dist/main.js",
     "lint": "eslint src",
     "lint:fix": "pnpm lint -- --fix",
+    "lint:openapi": "spectral lint http://localhost:${PORT:-3000}/openapi.yaml",
```

Comment:
The `${PORT:-3000}` bash syntax evaluates the provided `PORT` environment variable which is custom in CI, and falls back to `3000` for local development when none is explicitly provided.